index.html

```diff
@@ -2144,7 +2161,7 @@ <h3>Media Types</h3>
                     semantics for its elements (e.g., XML, JSON, CBOR). 
                     <span class="rfc2119-assertion" id="arch-schema">Thus,
                         the corresponding Interaction Affordances SHOULD
-                        declare a data schema to provide more detailed
+                        declare a <em>data schema</em> to provide more detailed
                         semantic metadata for the data exchanged.</span>
```

Comment:
Could you fix this to "syntactic metadata", please?

Reply:
good catch